Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -61,6 +61,7 @@ class BrowserViewController: UIViewController,
     var overlayManager: OverlayModeManager
     var appAuthenticator: AppAuthenticationProtocol
     var contextHintVC: ContextualHintViewController
```

Comment:
Can we rename this `ContextualHintViewController`? Since we now have two it seems the name doesn't fit well anymore
```suggestion
    var toolbarContextHintVC: ContextualHintViewController
```